### DISCUSSION TITLE
Removing controls from leafmap.Map?

### DISCUSSION THREAD

**Discussion by glemoine62** [opengeos/leafmap#943] [Q&A] [answered]
What's the syntax for removing controls from the leafmap.Map()?
I need to make some space on the map canvas (e.g. to put up a legend) and, since I do not need the DrawControl, I figured out the following:

```
m = leafmap.Map()
for c in m.controls:
    if type(c).__name__== 'DrawControl':
        m.remove(c)

```

this works fine, but seems a little clunky. Is there a more elegant syntax?

Guido

**Comment by giswqs** [comment, marked as answer]
m = leafmap.Map(draw_control=False)

**Reply by glemoine62** [reply]
Thanks, better indeed to exclude at Map creation, but I found also this:

m.remove(m.draw_control)

if you want to remove at a later stage.
